### PR TITLE
CIRC-8704 Add broker cluster documentation

### PR DESCRIPTION
**Pull request by esproul**

### CODE REVIEW

File: content/circonus/integrations/brokers/installation.md [hunk 1]
```diff
@@ -252,6 +250,49 @@ Use caution.  If a CN is specified that is already in use then the broker will r
...
+a cluster of one node). Responsibility for running a given check is delegated
+to one of the cluster nodes. If that node becomes unavailable, another cluster
+node will take over responsibility for running that check until the primary
+owner is once again available.
+
+The type of check(s) assigned to a multi-node cluster may necessitate
+additional infrastructure and configuration. For example, any check type that
+relies on metrics being pushed to a broker (e.g.,
+[agent checks](/circonus/integrations/agents),
+[HTTPTrap](/circonus/integrations/library/json-push-httptrap)) will require a
```
Comment (bdunavant) [inline content/circonus/integrations/brokers/installation.md:266]:
```suggestion
[HTTPTrap](/circonus/integrations/library/json-push-httptrap), and others) will require a
```

Comment (bdunavant) [reply content/circonus/integrations/brokers/installation.md:266]: Technically pull checks where agents are running in reverse mode also need this, but probably a bit too detailed for this part of the doc.

File: content/circonus/integrations/brokers/installation.md [hunk 2]
```diff
@@ -252,6 +250,49 @@ Use caution.  If a CN is specified that is already in use then the broker will r
...
+load balancer in front of the broker cluster in order to spread the load across
+all cluster nodes. Cluster nodes where such check types are used should set the
+`EXTERNAL_HOST` and `EXTERNAL_PORT`
+[environment variables](#environment-variables) to the IPv4 address and port of
+the load balancer.
+
+No special configuration is required for check types that pull metrics, such as
```
Comment (bdunavant) [inline content/circonus/integrations/brokers/installation.md:273]: Technically pull checks where agents are running in reverse mode also need this so they can always find a broker to figure out who to setup the reverse connection to.

File: content/circonus/integrations/brokers/installation.md (hunk 1 above)
Comment (esproul) [reply content/circonus/integrations/brokers/installation.md:266]: The `e.g.` means this is not an exhaustive list, but examples. Seems redundant.

File: content/circonus/integrations/brokers/installation.md (hunk 2 above)
Comment (esproul) [reply content/circonus/integrations/brokers/installation.md:273]: Is this the common case? If we think it's important to mention, let's work up some language for that. I'm not sure how to word it.

Comment (bdunavant) [reply content/circonus/integrations/brokers/installation.md:273]: I know it is a mode and that it still exists.  I cannot speak to how common it is in deployment.  Maybe just a small mention like.
`that pull metrics, unless using agents in reverse-mode to reach out to brokers.`

If you're technical enough to be using reverse-mode brokers, the reason why should be obvious and they can follow the earlier instructions.

File: content/circonus/integrations/brokers/installation.md [hunk 3]
```diff
@@ -259,16 +259,18 @@ to one of the cluster nodes. If that node becomes unavailable, another cluster
...
+For example, any check type that relies on metrics being pushed to a broker
+(e.g., [agent checks](/circonus/integrations/agents),
+[HTTPTrap](/circonus/integrations/library/json-push-httptrap), and others) will
+require a load balancer in front of the broker cluster in order to spread the
+load across all cluster nodes. Cluster nodes where such check types are used
+should set the `EXTERNAL_HOST` and `EXTERNAL_PORT`
 [environment variables](#environment-variables) to the IPv4 address and port of
-the load balancer.
+the load balancer. Likewise, such additional configuration is needed where
+Circonus agents operate in "reverse mode", reaching out to a broker to push its
```
Comment (bdunavant) [inline content/circonus/integrations/brokers/installation.md]: This is not accurate.   Reverse mode does not change things into a push model.    Reverse-mode agents are like reverse-mode brokers.    They are responsible for establishing a secure connection, and the broker/stratcon pull the data through this connection.

File: content/circonus/integrations/brokers/installation.md (hunk 2 above)
Comment (esproul) [reply content/circonus/integrations/brokers/installation.md:274]: Agreed, and have put in a statement regarding this.

**Pull request merged**